### PR TITLE
[Enhancement](bvar) Add bvar for bloom filter and segment

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

Add bvar for bloom filter and segment.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run performance

File: be/src/olap/rowset/segment_v2/bloom_filter.h [hunk 1]
```diff
@@ -181,6 +208,8 @@ class BloomFilter {
     uint32_t _size;
     // last byte's pointer in data for null flag
     bool* _has_null = nullptr;
+    // is this bf used for write
```
Comment (zhannngchen) [inline be/src/olap/rowset/segment_v2/bloom_filter.h:211]: The variable is only used for debugging, you should declear it in comment

File: be/src/olap/rowset/segment_v2/bloom_filter.h [hunk 2]
```diff
@@ -31,6 +32,14 @@
 namespace doris {
 namespace segment_v2 {
 
+inline bvar::Adder<size_t> g_total_bloom_filter_num("doris_total_bloom_filter_num");
```
Comment (zhannngchen) [inline be/src/olap/rowset/segment_v2/bloom_filter.h:35]: pls also add a bvar to record the bloomfilter created by pk index

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**